src/Avalonia.Native/WindowImpl.cs

```diff
@@ -138,10 +150,17 @@ protected override bool ChromeHitTest (RawPointerEventArgs e)
                     {
                         if (_doubleClickHelper.IsDoubleClick(e.Timestamp, e.Position))
                         {
-                            if (_canResize)
```

Comment:
I find it weird that we have potentially backend specific logic in the C# part of the Avalonia.Native. Usually, C# part of this project is just a common abstraction, thin layer between native code and the rest of Avalonia. Since _in theory_ Avaloina.Native can be used with not just macOS, but any native implementation of the interface.

Reply:
But that's not for this PR to think about.